virt/virtual_machines/advanced_vm_management/virt-dedicated-resources-vm.adoc

```diff
@@ -4,16 +4,15 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-dedicated-resources-vm
 toc::[]
 
-Virtual machines can have resources of a node, such as CPU,
-dedicated to them in order to improve performance.
+Virtual machines can have resources of a node, such as CPU, dedicated to them in order to improve performance.
 
 include::modules/virt-about-dedicated-resources.adoc[leveloffset=+1]
 
 == Prerequisites
 
-* The
-xref:../../../scalability_and_performance/using-cpu-manager.adoc[CPU Manager]  must
-be configured on the node. Verify that the node has the `cpumanager` = `true`
```

Comment:
This isn't related to your PR, but I think it's odd that the equals sign is left out of the backticks here. I think `cpumanager = true` would make more sense.